{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/client/templates/drf-yasg/swagger-ui.html

```diff
@@ -0,0 +1,18 @@
+{% extends "drf-yasg/swagger-ui.html" %}
+
+{% block extra_scripts %}
+    <script>
+        let authorization = {
+            Token: {
+                name: "Token",
+                schema: {
+                    type: "apiKey",
+                    "in": "header",
+                },
+                value: "Token {{ user.auth_token.key | safe }}"
+            }
+        }
+
+        localStorage.setItem(KEY_AUTH, JSON.stringify(Immutable.fromJS(authorization)))
```

Comment:
Where does `KEY_AUTH` come from? Guessing it's a global defined somewhere in `swagger-ui.html`...

Reply:
yea this is the logic Brenna figured out for Savi to inject the API key into the user's browser so it gets picked up by Swagger.
I don't actually know where this comes from or if there is a better solution. I just know that it works.